app/bundles/ApiBundle/Helper/PointEventHelper.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Mautic\ApiBundle\Helper;
+
+use Mautic\ApiBundle\Event\ApiEvent;
+
+class PointEventHelper {
+	public static function validateApiCall(ApiEvent $eventDetails, $action) {
+
+		if((string) $eventDetails->getIdRule() === (string) $action['id']) {
```

Comment:
@npracht `if` should have a space before the `(`

And comparing `===` which compares the types when you retype both variables to `(string)` before kind of doesn't make sense.